isovar/read_at_locus.py

```diff
@@ -93,131 +232,30 @@ def read_at_locus_generator(
     # We get a pileup at the base before the variant and then check to make sure
     # that reads also overlap the reference position after the variant.
     #
-    # Annoyingly AlignmentFile.pileup takes base-0 intervals but returns
-    # columns with base-1 positions.
+    # TODO: I want to pass truncate=True, stepper="all"
+    # but for some reason I get this error:
+    #      pileup() got an unexpected keyword argument 'truncate'
+    # ...even though these options are listed in the docs for pysam 0.9.0
     for column in samfile.pileup(
             chromosome,
-            base0_position_before_variant,
-            base0_position_before_variant + 1):
-        if column.pos != base1_position_before_variant:
+            start=base0_position_before_variant,
+            end=base0_position_after_variant + 1):
+
+        if column.pos != base0_position_before_variant:
```

Comment:
What's happening here? You want a range of pileups but then only the centered one? Should you ask for it directly?